modules/ROOT/pages/con_pipelines-concepts.adoc

```diff
@@ -1,17 +1,17 @@
 [id='pipelines-concepts_{context}']
 = Pipelines concepts
 
-The Custom Resource Definitions (CRDs) provided by OpenShift Pipelines include: `Task`,`Pipeline`,`PipelineRun`, `TaskRun`, and `PipelineResource`.
+Openshift Pipelines provide the following set of standard CRDs that act as the building blocks from which you can assemble a CI/CD pipeline for your application:
 
-* A `Task` is the smallest configurable unit in a `Pipeline` and is essentially a function of inputs and outputs that inform the `Pipeline` build. It can run individually or as a part of a `Pipeline`. A `Pipeline` includes one or more `Tasks`, with a `Task` consisting of one or more steps. Steps are a series of commands that are sequentially executed by the `Task`.
+* _Task_ is the smallest configurable unit in a Pipeline. It is essentially a function of inputs and outputs that form the Pipeline build. It can run individually or as a part of a Pipeline. A Pipeline includes one or more Tasks, where each Task consists of one or more steps. Steps are a series of commands that are sequentially executed by the Task.
```

Comment:
As discussed earlier, use Definition or labeled list. Search for it here: https://redhat-documentation.github.io/asciidoc-markup-conventions/

Reply:
The order specified here is in sequence in which they are created/executed. Also, the definitions are inter related. For example, TaskRun is defined based on the Pipeline and Pipeline run mentioned before.